src/context/reducers/notificationReducer.tsx

```diff
@@ -10,40 +10,44 @@ export enum NotificationItemVariant {
 }
 
 export interface NotificationItem {
-  variant: 'success' | 'danger' | 'warning' | 'info' | 'default'
-  title: string;
-  message: string;
-  timestamp: Moment;
+  variant: 'success' | 'danger' | 'warning' | 'info' | 'default', //here need? how does it work? what's it doing? remove?
+  title: string,
+  message: string,
+  timestamp: Moment
 }
 
 export type NotificationState = NotificationItem[];
 
 export const initialNotificationsState: NotificationState = [];
-
-// { 'variant': 'info', 'title': 'Unread info notification title', 'message': 'This is an info notification description', 'timestamp': moment().subtract(1, 'days') },
-// { 'variant': 'warning', 'title': 'Unread warning notification title', 'message': 'This is a warning!!', 'timestamp': moment().subtract(5, 'minutes') },
-// { 'variant': 'danger', 'title': 'Unread danger notification title', 'message': 'This is dangerous...', 'timestamp': moment().subtract(1, 'hour') },
-
 interface NotificationPayload {
   [NotificationActionTypes.SEND]: {
     notifications: NotificationItem[]
+  },
+  [NotificationActionTypes.CLEAR]: {},
+  [NotificationActionTypes.REMOVE]: {
+    notifications: NotificationItem[]
```

Comment:
See https://github.com/darwinai/covidnet_ui/pull/47/files?file-filters%5B%5D=.tsx&hide-deleted-files=true#r616341971
```suggestion
    index: number
```